src/__Private/codegen/CodegenRelations.hack

```diff
@@ -368,5 +368,7 @@ final class CodegenRelations extends CodegenBase {
     // https://github.com/hhvm/hhast/issues/151
     $_ = (this $x) ==> $x->flatten(dict[]) ?as this;
     $_ = (this $x) ==> $x->flatten(dict[]) as this;
+    /* HH_IGNORE_ERROR[4223] Only allowed in partial mode. */
+    (...) ==> null;
```

Comment:
https://github.com/facebook/hhvm/commit/ec4779427a1d786dd4ed820a2c82223465b92109 removed these from the codebase but the parser still supports it so I guess HHAST should too.